Updated Local SSD GiB sizes for Dv3

AndyHerb opened the pull request:
Updated Local SSD sizes based on blog post size announcement: https://azure.microsoft.com/en-gb/blog/introducing-the-new-dv3-and-ev3-vm-sizes/

azurecla commented:
Hi __@oranms__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (oranbr). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

AndyHerb committed:
Updated Local SSD GiB sizes for Dv3

Updated Local SSD sizes based on blog post size announcement: https://azure.microsoft.com/en-gb/blog/introducing-the-new-dv3-and-ev3-vm-sizes/
